playground/infrastructure/api/v1/api_pb2_grpc.py

```diff
@@ -302,6 +370,40 @@ def GetRunError(request,
             options, channel_credentials,
             insecure, call_credentials, compression, wait_for_ready, timeout, metadata)
 
+    @staticmethod
+    def GetValidationOutput(request,
```

Comment:
How default field in CI-CD related to validation/preparation output?

Reply:
@daria-malkova I believe that this appeared due to the fact that we have not updated the python proto files for a long time.

Reply:
ok